src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -33,11 +34,17 @@ fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelleList(): List<Oppfolgingsti
                 if (notSykedagSinceLastSykedagCounter > 0) {
                     // Only count Feriedag if at least one Arbeidsdag since last Sykedag
                     notSykedagSinceLastSykedagCounter++
+                } else if (sykedagerCounter > 0) {
+                    // Counts as Sykedag if at least one Sykedag before
+                    oppfolgingstilfelleSykedagList.add(it)
+                    sykedagerCounter++
+                    notSykedagSinceLastSykedagCounter = 0
```

Comment:
Jeg tror jeg skjønner dette, men fint med en gjennomgang til uken 😸

Reply:
Kanskje litt misvisende navn på sykedagerCounter - den brukes egentlig ikke til å telle sykedager, men for å bestemme hvilke feriedager som skal telle som sykedager. Antall sykedager bestemmes av størrelsen på oppfolgingstilfelleSykedagList.

Reply:
Update: Har refaktorert litt for å gjøre koden litt mindre forvirrende - se siste commit.